tests/validator/test_validator.py

```diff
@@ -589,3 +591,53 @@ def test_validator_set_active_batch(multi_batch_taxi_validator):
     assert multi_batch_taxi_validator.expect_column_values_to_be_between(
         "pickup_datetime", min_value=jan_min_date, parse_strings_as_datetimes=True
     ).success
+
+
```

Comment:
Without a test for what happens to the `active_batch_id` field of the `Validator` object, we would not know whether or not it is being set in the desired way...

Reply:
Good point! I'm not sure whether there is a way to test it in progress, but I I'll add a test case to make sure that the end-state is as expected.